edxval/models.py

```diff
@@ -659,20 +659,17 @@ class Meta:
         max_length=20,
         choices=TranscriptProviderType.CHOICES,
     )
-    # TODO remove exists in step 3 of renaming.
-    exists = models.BooleanField(default=False, help_text=u'Transcript credentials state')
```

Comment:
>Not sure why the migrations are appearing in this, or why it reports conflicts despite rebasing

Perhaps your rebase was not successful?  This line should have been NullBooleanField if it was rebased.